tests/features/serialization/GH4859.test.ts

```diff
@@ -0,0 +1,55 @@
+import { Entity,  PrimaryKey, Property, serialize } from '@mikro-orm/core';
+import { MikroORM } from '@mikro-orm/sqlite';
+
+@Entity()
+export class TimeSeriesEntityTest {
```

Comment:
no need to export things from the tests
```suggestion
class TimeSeriesEntityTest {
```